optimize/backend/src/it/java/io/camunda/optimize/AbstractCCSMIT.java

```diff
@@ -338,11 +362,34 @@ protected OffsetDateTime getTimestampForZeebeAssignEvents(
         Instant.ofEpochMilli(startOfElement.getTimestamp()), ZoneId.systemDefault());
   }
 
+  protected OffsetDateTime getTimestampForZeebeLastAssignEvents(
+      final List<? extends ZeebeRecordDto> eventsForElement, final String assigneeId) {
+    final ZeebeRecordDto startOfElement =
+        eventsForElement.stream()
+            .filter(
+                event ->
+                    event.getIntent().equals(ASSIGNED)
+                        && ((ZeebeUserTaskRecordDto) event)
+                            .getValue()
+                            .getAssignee()
+                            .equals(assigneeId))
+            .sorted(Comparator.comparing(ZeebeRecordDto::getTimestamp))
+            .reduce((first, second) -> second)
+            .orElseThrow(eventNotFoundExceptionSupplier);
+    return OffsetDateTime.ofInstant(
+        Instant.ofEpochMilli(startOfElement.getTimestamp()), ZoneId.systemDefault());
+  }
+
   protected OffsetDateTime getTimestampForZeebeUnassignEvent(
       final List<? extends ZeebeRecordDto> eventsForElement) {
     return getTimestampForZeebeAssignEvents(eventsForElement, "");
   }
 
+  protected OffsetDateTime getTimestampForZeebeLastUnassignEvent(
+      final List<? extends ZeebeRecordDto> eventsForElement) {
+    return getTimestampForZeebeLastAssignEvents(eventsForElement, "");
+  }
```

Comment:
Also not sure that this wrapper method is super helpful, as the called method is quite indicative of behaviour already